manifests/profiles/default.yaml

```diff
@@ -64,6 +64,12 @@ spec:
             - port: 443
               targetPort: 8443
               name: https
+            - port: 15012
```

Comment:
@howardjohn @costinm do we have any concerns about adding these ports by default?

Reply:
My only concern is if it breaks some LB. GCP ILB used to have 5 port limit which this exceeds, its now removed though.

@nrjpoddar @linsun any idea of other constraints?

Reply:
Note this *should* be temporary as hopefully we are all on port 443 in the near(ish) future

Reply:
This certainly simplifies the process for exposing istiod (no kubectl patch or bouncing ingress pods required). My concern was also WRT port limits and such. Do you have a rough timeline for the 443 solution?

Reply:
I could also remove 15017 and add that only in the central istio integration test through `ControlPlaneValues`

Reply:
Ok with this, thanks for asking @howardjohn.  I 'm basically running with 15012 and 15017 open before we can figure out switching to use 443.

Reply:
Don't you also need to add 15017?

Reply:
re https://github.com/istio/istio/pull/26146#issuecomment-669775514, I wanted to just add 15012 first to be safer. 15017 could be added easily later if needed